examples/volume.go

```diff
@@ -48,9 +47,32 @@ func main() {
 
 	if err != nil {
 		fmt.Printf("Failed to create volume, err ", err)
-		return
```

Comment:
Why is the return statement removed?  If volume creation fails, the subsequent actions will presumably also fail.